src/v2/view-builder/views/DeviceEnrollmentTerminalView.js

```diff
@@ -0,0 +1,58 @@
+import { loc } from 'okta';
+import BaseHeader from '../internals/BaseHeader';
+import HeaderBeacon from '../components/HeaderBeacon';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import OdaOktaVerifyTerminalView from '../components/OdaOktaVerifyTerminalView';
+import MdmOktaVerifyTerminalView from '../components/MdmOktaVerifyTerminalView';
+import Enums from 'util/Enums';
+
+const Header = BaseHeader.extend({
+  HeaderBeacon: HeaderBeacon.extend({
+    getBeaconClassName: () => 'mfa-okta-verify',
+  }),
+  initialize () {
+    const deviceEnrollment = this.options.appState.get('deviceEnrollment');
```

Comment:
any possibilities if `deviceEnrollment` can be null ?

Reply:
@yannongli-okta is there a chance api will return null or undefined here?